README.md

```diff
@@ -3,9 +3,8 @@
 
 
 <div align="center">
-<img src="http://www.shaghighi.ir/pyrgg/images/pyrgg-logo.png" height=240px width=320px>
-<hr/>
-<h1>Random Graph Generator</h1>
+<img src="https://github.com/sepandhaghighi/pyrgg/raw/master/otherfile/logo.png" width="450">
```

Comment:
`450px`

Reply:
It's a standard format for `img` tag `width` attribute.
Take a look at [this](https://www.w3schools.com/tags/att_img_width.asp) link